types/transaction.go

```diff
@@ -88,6 +88,14 @@ type Transaction interface {
 	// Remember to check for a possible intervention.
 	ProcessRequestBody() (*Interruption, error)
 
+	// AppendRequestBody attempts to write data into the body up to the buffer limit and
+	// returns an interruption of the body is bigger than the limit and the action is to
+	// reject. This is specially convenient to resolve an interruption before copying
+	// the body into the RequestBodyWriter.
+	//
+	// It returns the corresponding interruption, the number of bytes written an error if any.
+	AppendRequestBody(io.Reader) (*Interruption, int, error)
```

Comment:
Let's remove `RequestBodyWriter` since otherwise there will be too many ways to do the same thing and make the code hard to manage

Reply:
That is the plan